core/src/com/group66/game/BustaMove.java

```diff
@@ -22,6 +22,21 @@
     /** The logger. */
     public static Logger logger;
     
+    /** Create the only object of this class */
+    private static BustaMove game = new BustaMove();
+   
+    private BustaMove() {   
+    }
+    
+    /**
+     * Gets the only object available.
+     * 
+     * @return the only available object
+     */
+    public static BustaMove getGameInstance() {
+        return game;
+    }
```

Comment:
I made it just like in slide 11 in the DP2 lecture. This is another way to do it. Three ways can be used to describe the Singleton pattern, according to the lecture.